lib/core/src/display/rendering/object.rs

```diff
@@ -0,0 +1,47 @@
+use super::Transform;
+
+/// Base structure for representing a 3D object in a `Scene`
+pub struct Object {
```

Comment:
I like to simply call it Object because it's going to represent an object (like the real world thing we all are used to) in a virtual space.

Do you think calling it Object might be confusing?